modules/docs/lib/widgets/enhancedComponent.tsx

```diff
@@ -68,7 +68,17 @@ registerWidget<EnhancedComponentValue>('enhancedComponent', ({value, doc, meta})
     <>
       {value.styleComponent ? (
         <>
-          <Heading headingOffset={1}>Layout Component</Heading>
+          {console.log(parentComponentName)}
+          <Heading
```

Comment:
Got an extra `console.log` here.
```suggestion
          <Heading
```